### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by emilyseibert**
* first stab at local state on the front end vs. app state
* first stab at styles 
* reorg'd some of the container logic 
* Navbar won't match Designs because of limited V1 views; this version has been approved by jesse

### CODE REVIEW

**Comment by emilyseibert** [comment]
closes #12

**Comment by emilyseibert** [comment]
closes #52

File: less/components/navbar.less [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import "../colors";
```
Comment (oconn) [inline less/components/navbar.less]: Nice! I love that you decided to go with less

File: src/cljs/wombats_web_client/components/navbar.cljs [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
+(def selected (reagent/atom "open-games"))
+
 (defn login []
   [:a {:href github-signin-url} "Login"])
 
+(defn wombat-logo []
+  [:a {:href "#/"} [:img.wombat-logo {:src "/images/img-logo-horizontal.svg"}]])
+
+(defn nav-link
+  [id class onClick link title]
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/navbar.cljs]: It's more common to pass maps when there are more than a few associated params and use destructuring.

`{:keys [id class on-click link title]}`

File: src/cljs/wombats_web_client/components/navbar.cljs [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
+  [:li {:id id :class class :on-click onClick} [:a {:class (if (= @selected id) "active") :href link} title]])
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/navbar.cljs]: spacing on this is a little hard to read. It seems to be more common to format the above like

```clj
[:li {:id id 
      :class class 
      :on-click onClick} 
  [:a {:class (if (= @selected id) "active") 
       :href link} title]])
```

**Comment by emilyseibert** [comment]
@oconn made revisions. haven't done too much destructuring so hoping that looks right

**Pull request merged**